OpenRA.Mods.Common/Traits/SpawnActorOnDeath.cs

```diff
@@ -90,6 +94,10 @@ void INotifyRemovedFromWorld.RemovedFromWorld(Actor self)
 			if (attackingPlayer == null)
 				return;
 
+			var defeated = self.Owner.WinState != WinState.Undefined;
+			if (defeated && Info.SpawnAfterDefeat)
```

Comment:
The check here is inverted.  We want to suppress the husks when `SpawnAfterDefeat` is false.